src/app/app.component.ts

```diff
@@ -1,11 +1,11 @@
 import { Component } from '@angular/core';
-import { UsersService } from './users.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'Angular-practice';
+  title = 'RoutingServiceDemo';
```

Comment:
If this variable `title` is not being used, then it can be removed

Reply:
Renamed the title to UserManagementSystem